index.html

```diff
@@ -0,0 +1,536 @@
+            send payouts, and manage their businesses online.
+          </p>
+          <div class="mt-6 space-x-4">
+            <button
+              class="bg-slate-800 text-white text-sm px-6 py-2 rounded-full hover:bg-slate-700"
+            >
+              Start now
+            </button>
+            <button
+              class="text-slate-800 text-sm font-bold hover:text-slate-600"
+            >
+              Contact sales
+            </button>
+          </div>
+        </div>
+
+        <div class="hidden md:block relative -mr-35">
+          <img class="" src="assets/dashboard.png" alt="" />
+          <img class="absolute top-16 -left-24" src="assets/phone.png" alt="" />
+        </div>
+      </div>
+    </div>
+
+    <!-- starting of partner-->
+    <div class="md:max-w-container mx-auto">
+      <div class="mt-16 md:mt-32 px-10 grid grid-cols-2 md:grid-cols-4 gap-20">
+        <img src="assets/logo1.svg" alt="" />
+        <img src="assets/logo2.svg" alt="" />
+        <img src="assets/logo3.svg" alt="" />
+        <img src="assets/logo4.svg" alt="" />
+        <img src="assets/logo5.svg" alt="" />
+        <img src="assets/logo6.svg" alt="" />
+        <img src="assets/logo7.svg" alt="" />
+        <img src="assets/logo8.svg" alt="" />
+      </div>
+    </div>
+
+    <!-- starting of Unified platform-->
+    <div class="bg-second mt-28">
+      <div
+        class="max-w-container mx-auto px-4 pt-32 grid md:grid-cols-2 gap-10"
+      >
+        <p class="text-lg font-bold text-indigo-500">Unified platform</p>
+        <h1
+          class="text-4xl text-slate-800 font-bold leading-snug tracking-wide col-start-1"
+        >
+          A fully integrated suite of payments products
+        </h1>
+        <p class="text-gray-500 font-medium col-start-1">
+          We bring together everything that’s required to build websites and
+          apps that accept payments and send payouts globally. Stripe’s products
+          power payments for
+          <span class="text-indigo-500 hover:text-black cursor-pointer">
+            online and in-person retailers,</span
+          >
+          <span class="text-indigo-500 hover:text-black cursor-pointer"
+            >subscriptions businesses,</span
+          >
+          <span class="text-indigo-500 hover:text-black cursor-pointer"
+            >software platforms and marketplaces,</span
+          >
+          and everything in between.
+        </p>
+
+        <p class="text-gray-500 font-medium">
+          We also help companies beat fraud, send invoices, issue virtual and
+          physical cards, reduce friction at checkout, get financing, manage
+          business spend, and much more.
+        </p>
+        <button
+          class="w-max bg-indigo-500 text-sm text-white px-7 py-2 rounded-full hover:bg-slate-800"
+        >
+          Start with payment
+        </button>
+      </div>
+      <!-- => starting of Unified platform Gallery-->
```

Comment:
yours
<img width="374" alt="Screen Shot 2022-11-22 at 09 28 21" src="https://user-images.githubusercontent.com/94446528/203252296-f57ce3cf-dc19-45af-ad25-ddc2c9955096.png">

original
<img width="374" alt="Screen Shot 2022-11-22 at 09 28 29" src="https://user-images.githubusercontent.com/94446528/203252358-1a16b3ef-569b-4b91-a5d7-ee4b7c37368b.png">


try to order well those images